docs/client.md

```diff
@@ -0,0 +1,53 @@
+# pb-cli specification
+
+## Overview
+
+`pb-cli` is a command line tool which communicate with `pb-go` backend. Although user can just upload data to pastebin directly with `curl`, we still provide a tools to help user easily use our advanced feature such as burn-after-reading and private share.
+
+We guarantee everything that worked with `pb-cli`, also worked with `curl`.
+
+## Configuration
+
+You can use `-c file` to provide a config file. If you didn't provide one, it will find config file called `.pb-cli.toml` from current directory then home directory.
+
+Config file should be written in `toml` like:
+
+```toml
+host = "http://your.pastebin.bakcend:port"
```

Comment:
tell user, if port is default, like 443, it can be ignored.

Since we have no guarantee about transport layer safety, please do change all http scheme to https instead.